packages/checkbox-group/src/vaadin-checkbox-group.js

```diff
@@ -370,9 +370,8 @@ class CheckboxGroup extends FieldMixin(FocusMixin(DisabledMixin(ElementMixin(The
   _setFocused(focused) {
     super._setFocused(focused);
 
-    // Do not validate when focusout is caused by document
-    // losing focus, which happens on browser tab switch.
-    if (!focused && document.hasFocus()) {
```

Comment:
The [original issue](https://github.com/vaadin/flow-components/issues/5181), for which this `hasFocus()` check was added, reported that there was unintended validation on blur once you switch to another browser tab in the case you focused the field but hasn't typed anything.